ZenPacks/ZenSystems/ApcUps/zenpack.yaml

```diff
@@ -0,0 +1,169 @@
+name: ZenPacks.ZenSystems.ApcUps
+device_classes:
+  /Power/UPS/ApcUps:
+    create: True
+    remove: True
+    zProperties:
+      zCollectorPlugins: ['community.snmp.ApcUpsDeviceMap', 'community.snmp.ApcUpsBatteryMap']
+      zDeviceTemplates: ['ApcUps', 'ApcUpsAdvanced', 'Device']
+      zPythonClass: ZenPacks.ZenSystems.ApcUps.ApcUpsDevice
+      zSnmpVer: v1
+
+    templates:
+      ApcUps:
+        description: ''
+        targetPythonClass: Products.ZenModel.Device
+        datasources:
+          actualVoltage:
+            type: SNMP
+            datapoints:
+              actualVoltage:
+                aliases: {actualVoltage: null}
```

Comment:
Let's find the units and append them with aliasName__units, for example, actualVoltage__volts